custom/icds/location_reassignment/views.py

```diff
@@ -211,11 +214,10 @@ def _process_request_for_household_reassignment(self, parser, request, uploaded_
 def download_location_reassignment_template(request, domain):
     location_id = request.GET.get('location_id')
 
-    if not location_id:
+    if not location_id or not user_can_access_location_id(domain, request.couch_user, location_id):
```

Comment:
Ya. I assumed location search already handled that since its already used at so many places.
This is where i believe it handles that
https://github.com/dimagi/commcare-hq/blob/d2e1b01025435acbd33ed986c16ca9932b2bb68c/corehq/apps/reports/filters/controllers.py#L94

Reply:
sweet